buildScripts/compendium-pack.ts

```diff
@@ -184,3 +185,47 @@ function lookup(dict: any, key: string): string {
 export function isObject(value: unknown): boolean {
   return typeof value === "object" && value !== null;
 }
+
+function includePackYaml(yamlString: string): string {
+  const tokenRegex = /\"\|{{\s*([\w+.\/-]+)\s+([\w.-]+)\s+\[([^\[]*)\]\s+}}\|\"/gm;
+
+  const match = yamlString.match(tokenRegex);
+
+  const yaml = yamlString.replace(tokenRegex, function (match: string, ...args) {
+    let packName = args[0];
+    let rqid = args[1];
+    let overrides = args[2].split(",").map((item: string) => item.trim());
+
+    const packTemplate = path.resolve("./src/assets/pack-templates/" + packName);
+
+    let packTemplateYamlString = fs.readFileSync(packTemplate, "utf-8");
+
+    let packEntries = packTemplateYamlString.split("---");
+
+    for (const entry of packEntries) {
+      if (entry.includes(rqid)) {
+        // Add two spaces to every line to indent it properly
+        let cleanEntry = entry.replace(/\r/, "").replace(/[\n]/gm, "\n  ");
+
+        for (const override of overrides) {
+          const split = override.split(":").map((item: string) => item.trim());
+          const overrideRegex = new RegExp(`(${split[0]}:\\s*)(.)`, "m");
```

Comment:
Only tested in a regex tester, but shouldn't the last part of the regex be `(.*)` so you get all of the value and not just the first character? 

Also the `split[0]` should be escaped
```ts
const overrideRegex = new RegExp(`(${escapeRegex(split[0])}:\\s*)(.*)`, "m");
```

Reply:
Yep.